### ISSUE TITLE
Navigation editable list view transfers focus to canvas when editing a menu item

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#47607]
Currently when using a keyboard (or assitive tech) to use the Nav block's list view editing focus is not managed as you'd expect.

Clicking "Edit" on a node causes the block to be selected (good) and focus to be transferred to the editor canvas (bad).

The focus should be placed on the inspector controls for the selected block. There is currently no programmatic interface that allows us to achieve this. Therefore users are required to manually move focus to the inspector controls which results in a lot of context switching.

Moreover, once you have finished editing and you go "back" to the Nav block, the list view is not focused. Again this requires you to manually move focus back to the list view.

**Comment by jeryj** [comment]
![Editing navgivation from inspector sends focus to canvas](https://github.com/WordPress/gutenberg/assets/967608/3ae3d7b1-9dc6-4929-8000-cea1c61d8e20)

Is this gif showing the interaction you mean? When editing an item from the sidebar block tools, the focus should be sent to the form in the sidebar that opens, but instead it's being sent to the editor canvas?

**Comment by getdave** [comment]
@jeryj Yes that's it. It's because we have to `selectBlock()` to show the inspector controls. But that causes focus to jump into the canvas. 

Ideal solution:

- the block in the canvas is highlighted somehow
- the focus remains in the List View sidebar where editing was happening

I wonder if `selectBlock` has any options to cancel the auto-focusing of canvas?

**Comment by getdave** [comment]
This looks to be a duplicate of https://github.com/WordPress/gutenberg/issues/47018. I'm closing in favour of that Issue.

**Comment by getdave** [comment]
I was wrong. That one was about _Creating Menus_.
